docs/_includes/header.html

```diff
@@ -11,11 +11,26 @@
 <link rel="apple-touch-icon" sizes="144x144" href="/dist/images/apple-touch-icon-144x144.png">
 <link rel="apple-touch-icon" sizes="152x152" href="/dist/images/apple-touch-icon-152x152.png">
 <link rel="apple-touch-icon" sizes="180x180" href="/dist/images/apple-touch-icon-180x180.png">
-<link rel="icon" type="image/png" href="/dist/images/favicon-32x32.png" sizes="32x32">
 <link rel="icon" type="image/png" href="/dist/images/android-chrome-192x192.png" sizes="192x192">
+
+<!-- Default favicons -->
+<link rel="shortcut icon" type="image/x-icon" href="/dist/images/favicon.ico">
+<link rel="icon" type="image/png" href="/dist/images/favicon-32x32.png" sizes="32x32">
 <link rel="icon" type="image/png" href="/dist/images/favicon-96x96.png" sizes="96x96">
 <link rel="icon" type="image/png" href="/dist/images/favicon-16x16.png" sizes="16x16">
-<link rel="shortcut icon" type="image/x-icon" href="/dist/images/favicon.ico">
+
+<!-- Light mode favicons -->
```

Comment:
Do we need these light mode declarations if they're the default? Seems like we can just have the dark mode overrides.